### ISSUE TITLE
Add a copy and paste reference option for open windows 

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#687]
Hi again. 

I had another thought this morning which I thought was cool. Video attached explaining it. It addresses an issue I raised before in a better way. In brief, what do you think of adding new menu options to the window button that allows the user to copy a verse reference and then paste the reference. That way I could copy the reference in one window and paste it to any window of my choice. It's basically just a quick way of selecting a verse.

Hmm, adding two menu options to that popup menu would clutter it up a bit. You already have the long click option and then 'Copy Verse' from the menu. I use that a lot. That handles the copying.

So just need to find a way to to paste the verse reference into a window. It seems intuitive to attach that option to the window button. But another place would be a paste button in the verse selector itself. There is not much space in that selector so that might be hard.

Anyhow, I think this suggestion is better than my previous suggestions 😁 It is more flexible. 

And the video which I can't turn into a link 
https://drive.google.com/file/d/17NOqE4WzDxm5Is9jg7ZAddSJPttLV8x6/view?usp=drivesdk

**Comment by tuomas2** [comment]
Related: #506 #61 #597, #429, #232 .

Good idea. To be considered together with other related feature issues.

**Comment by agrogers** [comment]
Wow, so many ideas here that you are able to manage. You should be paid better I think.

**Comment by timbze** [comment]
I love the copy > paste idea. I have also been thinking of "Open in workspace" option.

My idea is that I see the verse at the top reference bar while I'm reading in my reading plan workspace, or "Find" workspace, and I want to open that verse in my "Study" workspace.

My suggestion is:
1. Long press reference at the top (currently that opens "Choose document")
2. Now it would open menu that you can "Choose document" or "Open in workspace"
3. Click "Open in workspace" > submenu with windows appears where you can click the windows you want to "Paste" the reference in.
4. Either the "Study" workspace opens with the passage I had selected to open, or a bar pops up at bottom, like when creating new bookmark, that says "Go to "Study" workspace", then I can just let it disappear or click if I had wanted to go there immediately.

**Comment by agrogers** [comment]
I didn't try hard enough to properly understand your suggestions 😏 But I
liked the idea of copying to another workspace.

My thought was that once you have copied a verse you could then paste it
into any other panel in any workspace.

The actual mechanics of that I'm not sure about. But I don't think I needed
it integrated to the workspace or verse selector.

I reckon the existing verse copy option could be fine. Just need the paste
option added somewhere.

**Comment by agrogers** [comment]
Hi Tuomas. I thought this was resolved. But alas not as I expected. The paste reference is not intended to paste the document as well. It should just change the reference for whatever the current document is. I think this is the right way to do it because you can always change the document as a second step. I think rarely would I want to copy the document as well - the document would already be set.

**Comment by tuomas2** [comment]
@agrogers are you referring to #2935? This issue is still open

**Comment by tuomas2** [comment]
Could create a bug ticket. For non-bibles doc need to be set too.

**Comment by agrogers** [comment]
Yes, referring to the feature you added for me. I understood pasting the
link would only paste the 'reference' (as per the toast message) not the
bible version as well. So when the reference is pasted into, say, a
commentary window the commentary will change to the reference. (I am using
'paste' where you use 'open')

Usually I only use one bible translation and usually want to paste a link
into a bible window. But sometimes I want to use the reference to change a
non linked commentary window (or another bible version) .

The menu does say 'Open NASB: xxx' but I didn't connect the dots.
